records_mover/db/bigquery/bigquery_db_driver.py

```diff
@@ -24,70 +22,14 @@ def __init__(self,
         super().__init__(db)
         self._bigquery_loader = BigQueryLoader(db=self.db, url_resolver=url_resolver)
 
-    def load(self,
-             schema: str,
-             table: str,
-             load_plan: RecordsLoadPlan,
-             directory: RecordsDirectory) -> int:
-        """Loads the data from the RecordsDirectory instance named
-        'directory'.  Guarantees a manifest file named 'manifest' is
-        written to the target directory pointing to the target
-        records.
-
-        Returns number of rows loaded (if database provides that
-        info).
-
-        """
-        return self._bigquery_loader.load(schema=schema,
-                                          table=table,
-                                          load_plan=load_plan,
-                                          directory=directory)
-
-    def load_from_fileobj(self, schema: str, table: str,
-                          load_plan: RecordsLoadPlan, fileobj: IO[bytes]) -> Optional[int]:
-        return self._bigquery_loader.load_from_fileobj(schema=schema,
-                                                       table=table,
-                                                       load_plan=load_plan,
-                                                       fileobj=fileobj)
-
-    def can_load_from_fileobjs(self) -> bool:
-        return True
-
-    def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool:
-        return self._bigquery_loader.can_load_this_format(source_records_format)
-
-    def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
-        return self._bigquery_loader.known_supported_records_formats_for_load()
```

Comment:
In the past, we'd implement the short loading/unloading-related functions directly, and delegate the longer ones, which leaves things in multiple places and generally bulks up the drivers, making them harder to get your head around.